meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -59,7 +59,7 @@
 #       a) [optional] Define RESIN_DEFCONFIG_NAME. Default: "resin-defconfig"
 #       b) Add RESIN_DEFCONFIG_NAME to SRC_URI.
 
-inherit kernel-resin
+inherit kernel-resin compress-kernel-modules
```

Comment:
I believe the "inherit kernel-resin" is a typo here. Maybe we can clean this while we are it?